lambda/recipes-responder/src/commandline-reindex.ts

```diff
@@ -150,15 +160,16 @@ async function main() {
 
   console.log("------------------------------------------------------");
   console.log("Rebuilding index...");
-  const indexData = await retrieveIndexData();
-  await writeIndexData(indexData);
+  const indexDataForAllRecipes = await retrieveIndexData();
+  await indexing(indexDataForAllRecipes, test)
+
```

Comment:
☝️ This needs to be updated to handle the split index - I messed around with it while testing and haven't fixed it up, so will do that next